.github/workflows/dash.yaml

```diff
@@ -0,0 +1,28 @@
+name: dash
+
+on:
+  pull_request
+
+jobs:
+  check-dash:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Create Dash Dependency Report
+        run: ./gradlew mergeDashFiles
+
+      - name: Dash license check
+        uses: ./.github/actions/check-dash
```

Comment:
Check and see if you can use `uses: eclipse-kuksa/kuksa-actions/check-dash@2`

like in https://github.com/eclipse/kuksa.val/blob/876f210c9d5d5e0595b927c2845b33bfcb9f76fb/.github/workflows/kuksa_databroker_build.yml#L240

If so we do not need to add check-dash in this repo.

Reply:
Good find! We also need to align this on the SDK-side as well, because the SDK also stores it's own dash-action right now.

Replaced it as suggested.

Reply:
The more we can reuse the better.